spec/spec.md

```diff
@@ -612,7 +662,13 @@ The following section details where the _Presentation Submission_ is to be embed
 ```
 :::
 
-#### DID Comms
+#### DIDComm
+
+DIDComm is lower level than the problem domain of this spec. It allows transmission of arbitrary content, so it is capable of carrying the data formats described here, in the same way you could transmit the format over TCP.
+
+What's required in a DIDComm-based approach to communication is a higher-level protocol that explains the steps in the "dance" of proving something. This is the subject of Aries RFC 0037. That protocol, or future DIDComm based protocols like it, could be used to send a proof request (what is here called a "presentation definition") and receive a verifiable presentation (which could be wrapped in the presentation submission described here). The protocol is deliberately defined to be data format agnostic, so it is directly usable with this data format without modifying either doc.
+
+TODO: I recommend that you draw a distinction between the definition of a presentation (here called a "presentation definition", very appropriately) and the request for such a presentation. It seems quite likely to me that presentation definitions will be standardized and published. For example, there might be a standard presentation definition that captures the requirements of people applying for citizenship in a given country. Once defined, the actual interactions where proofs are demanded need not define them again; they might just refer to them. Thus, it is possible to REQUEST a presentation without DEFINING it, by simply refering to a previously published DEFINITION. This is why the Aries protocol describes the challenge for proof as a "request", not a "definition" -- a request could include an inlined definition, or simply refer to one that's external to the request.
```

Comment:
We need concrete examples of the JSON representation of the messages required to support this spec over didcomm here.

Reply:
raised issue #52